### ISSUE TITLE
SV classification

### ISSUE THREAD

**Issue by minglibio** [vgteam/vg#3627]
Hi all,

I would like to know if anyone has ideas on how should I get the SV types (INS/DEL/INV) from the output vcf of minigraph>cactus>vg-giraffe>vg-call pipeline. Are there any tools I can use? Or should I classify them by myself? If so, what criteria should I use, especially for inversion?

Best,
Ming

**Comment by Chaima-Bouchenak** [comment]
Hello, 
did you try to manipulate your final VCF file with the vcf/bcf tools package ? it provides many commands for file exploration , hope this [link](https://eriqande.github.io/eca-bioinf-handbook/basic-handling-of-vcf-files.html) would be helpful.

**Comment by minglibio** [comment]
Hi @Chaima-Bouchenak 
Thanks for your suggestion. But I don't find the command of vcf/bcftools that can classify the SVs.

**Comment by Drbiology** [comment]
Hi,
Did you solve the problem, I also encountered the same problem. Can you give me some advice?
Thanks!

**Comment by dudududu12138** [comment]
Hi， I also encountered the same problem. Can you give me some advice?

**Comment by faithokamoto** [comment]
I do not believe we have any standard classification tool within vg itself. You could parse the VCF (many such tools available) and classify where deletions are a loss of sequence, insertions are a gain, and inversions are a reverse complement.

**Comment by faithokamoto** [comment]
Swapping to duplicate since it truthfully isn't there but we should add it.
